lottie/src/main/java/com/airbnb/lottie/model/content/GradientColor.java

```diff
@@ -28,6 +28,19 @@ public int getSize() {
   }
 
   public void lerp(GradientColor gc1, GradientColor gc2, float progress) {
+    // Fast return in case start and end is the same
+    // or if progress is at start/end or out of [0,1] bounds
+    if (gc1 == gc2) {
```

Comment:
This is java so if you want to use your equals method, you have to use .equals

Reply:
Oh my, so long ago... 🙈 
```suggestion
    if (gc1.equals(gc2)) {
```